### PR TITLE
Point in polygon tutorial

### PR DESCRIPTION
**Pull request by hansvancalster**
This PR implements a tutorial starting from #46

### CODE REVIEW

File: content/tutorials/spatial_point_in_polygon/index.Rmd [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+Next, we use `rgdal::readOGR` to connect with the sqlite/spatialite database and extract the Belgium 10 km x 10 km grid as a SpatialPolygonsDataFrame. Apart from the 10 km x 10 km grid, the database also contains 1 km x 1 km and 100 km x 100 km grids as raster or vector files.
+
+```{r}
+be10grid <- readOGR(dsn = paste0(tempdir(), "\\Belgium.sqlite"), 
+                  layer = "be_10km")
+```
+
+
+
+## Point in polygon with the `sp` package
```
Comment (florisvdh) [inline content/tutorials/spatial_point_in_polygon/index.Rmd]: Considering the preference for the `sf` package these days (tidyverse compatible etc.), I'd suggest to move this part to the end of the tutorial. Or even, split off into a separate sp-flavoured tutorial.

File: content/tutorials/spatial_point_in_polygon/index.Rmd [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+  st_transform(crs = 31370)
+```
+
+Now we are ready to make the spatial overlay. This is done with the aid of `sf::st_join`. We set the join type to `st_within` and join the information from the grid to the points through a left join. See [the DE-9IM topological model](https://en.wikipedia.org/wiki/DE-9IM) for explanations about all possible spatial joins. 
+
+
+```{r}
+invasive_be10grid <- invasive_spatial %>%
+  st_join(be10grid,
+          join = st_within,
```
Comment (florisvdh) [inline content/tutorials/spatial_point_in_polygon/index.Rmd]: I think the default setting `join = st_intersects` would do the job, no?

Comment (hansvancalster) [reply content/tutorials/spatial_point_in_polygon/index.Rmd]: Good point. But there is a difference. With ` sp::over`, points on a polygon boundary and points corresponding to a polygon vertex are considered to be inside the polygon. This corresponds with `st_intersects`, so I should have used that to get the same behavior. I will change this and note that `st_within` can be used if we want to join only when points are within (excluding boundary).

File: content/tutorials/spatial_point_in_polygon/index.Rmd (hunk 1 above)
Comment (hansvancalster) [reply content/tutorials/spatial_point_in_polygon/index.Rmd]: I agree. Will change this, but keeping the sp-way in the same tutorial.

File: content/tutorials/spatial_point_in_polygon/index.Rmd (hunk 2 above)
Comment (florisvdh) [reply content/tutorials/spatial_point_in_polygon/index.Rmd]: OK, clear!

**Comment by hansvancalster** [comment]
@peterdesmet something went wrong with the deployment of the website after I merged this PR. I looked at the travis CI log, but I don't understand it fully. Can you have a look at this?

**Comment by peterdesmet** [comment]
Is fixed. Looks like a token from @stijnvanhoey was used that is no longer valid. We'll have to find a better way to manage tokens, but for now I created a new one under my account:

https://travis-ci.org/inbo/tutorials/jobs/557840723#L322

**Comment by hansvancalster** [comment]
Great, thanks!

**Pull request merged**